Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -135,14 +135,17 @@
               <button mat-button [routerLink]="['/rooms']" i18n="@@navigationRooms">
                 Rooms
               </button>
-              <button mat-icon-button [matMenuTriggerFor]="personalmenu">
+              <button mat-icon-button [matMenuTriggerFor]="personalmenu" class="profilebutton">
                 <mat-icon
+                  *ngIf="currentUser == null"
                   matBadge="!"
                   matBadgeColor="accent"
                   matBadgeSize="small"
                   [matBadgeHidden]="!releaseInfoService.newUpdate"
                   >person</mat-icon
                 >
+                <app-profile-picture *ngIf="currentUser != null" class="pb-self-centered circle square-img40" [user]="currentUser" [altText]="currentUser.Id">

```

Comment:
We might need to add a more descriptive text for users with screen readers. currentUser.Id is probably not enough info to realize that this is a menu

Reply:
You can add screen reader text using `<span class="cdk-visually-hidden">the text</span>`. With an explanatory text the `alt` attribute of the component can be left empty.

Reply:
this might not be necessary because material provides correct a11y hints by default